website/docs/d/federated_settings_org_configs.html.markdown

```diff
@@ -41,12 +43,15 @@ In addition to all arguments above, the following attributes are exported:
 * `totalCount` - Count of the total number of items in the result set. It may be greater than the number of objects in the results array if the entire result set is paginated.
 
 ### FederatedSettingsOrgConfigs
-          
+
 * `domain_allow_list` - List that contains the approved domains from which organization users can log in.
 * `domain_restriction_enabled` - Flag that indicates whether domain restriction is enabled for the connected organization.
-* `identity_provider_id` - Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
+* `identity_provider_id` - Legacy 20-hexadecimal digit string that identifies the UI access identity provider that this connected org config is associated with. This id can be found within the Federation Management Console > Identity Providers tab by clicking the info icon in the IdP ID row of a configured identity provider.
```

Comment:
question: do we need to clarify that users perform this step in Atlas? I think it might be useful

Reply:
I updated the description in 31b4e040fbe74a4d67bb34a1227378e56d6c5527
WDYT?